packages/@react-spectrum/s2/src/ComboBox.tsx

```diff
@@ -392,6 +407,11 @@ export function ComboBoxItem(props: ComboBoxItemProps): ReactNode {
                     icon: {render: centerBaseline({slot: 'icon', styles: iconCenterWrapper}), styles: icon}
                   }
                 }],
+                [AvatarContext, {
+                  slots: {
+                    avatar: {size: avatarSize[size], styles: avatar}
```

Comment:
we don't need to centerBaseline this one? looks like the icon is

Reply:
Resolved by Devon's fix